crates/alkahest-renderer/src/ecs/utility.rs

```diff
@@ -121,22 +144,22 @@ impl Default for Beacon {
 
 impl Utility for Beacon {
     fn default_label() -> Label {
-        Label::new_default("Beacon")
+        Label::new_default("Beacon").with_offset(0.0, 0.0, -0.5)
     }
 
     fn icon() -> Icon {
         Icon::Unicode(ICON_SIGN_POLE)
     }
 }
 
-pub struct RouteNode {
+pub struct RouteNodeHolder {
```

Comment:
The whole route implementation is bloating the utility file up hard, I would prefer if we move it's implementation to a `route` module next to this one

Reply:
I've moved everything but the drawing stuff to its own file, since that would mean moving it out of draw_utilities_system.